tests_integration/__tests__/__snapshots__/plugin-options-string.js.snap

```diff
@@ -17,7 +17,16 @@ exports[` 1`] = `
                              How to handle whitespaces in HTML.
                              Defaults to css.
     --jsx-bracket-same-line  Put > on the last line instead of at a new line.
-                             Defaults to false."
+                             Defaults to false.
+    --jsx-single-quote       Use single quotes in JSX.
+                             Defaults to false.
+-   --parser <flow|babel|babel-flow|babel-ts|typescript|css|less|scss|json|json5|json-stringify|graphql|markdown|mdx|vue|yaml|html|angular|lwc>
++   --parser <flow|babel|babel-flow|babel-ts|typescript|css|less|scss|json|json5|json-stringify|graphql|markdown|mdx|vue|yaml|html|angular|lwc|foo-parser>
```

Comment:
I think we should display where this parser option from.

`--foo-string` need do that too.

Maybe seperate issue.

Reply:
We can add the name of the originating plugin to the description of the value. Now the language name is use for it:

https://github.com/prettier/prettier/blob/93bdc0c61e19223c8818415756711aea38c15310/src/main/support.js#L115 

Should we change it from e.g. "PHP" to "PHP (@prettier/plugin-php plugin)"?

Reply:
Sounds good to me.